usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -1066,11 +1070,22 @@
                     )
                 ),
                 TRUE
-            ) AS tas_components
+            ) AS tas_components,
+        CASE
+            WHEN SIZE(COLLECT_SET(rpa.program_activity_name)) > 0
+                THEN COLLECT_SET(rpa.program_activity_name)
```

Comment:
Should the resulting SET be sorted similar to other COLLECT_SET implementations?

Reply:
I don't think it matters if the SET is sorted or not. These fields will be searched in Elasticsearch and I don't know if Elasticsearch benefits from a sorted list to help cut down on searching the entire collection.

I'm picturing the Elasticsearch query looking like this pseudocode:
```python
if <user input> in program_activity_names or program_activity_codes:
    return this Award/Transaction/Subaward
```

Reply:
If I remember correctly this helps with the downloads where we provide some of these aggregated arrays in a single column.